sharding-orchestration/sharding-orchestration-core/src/test/resources/META-INF/services/org.apache.shardingsphere.orchestration.center.api.DistributedLockManagement

```diff
@@ -15,5 +15,6 @@
 # limitations under the License.
 #
 
-org.apache.shardingsphere.orchestration.temp.internal.registry.fixture.FirstTestRegistryCenter
-org.apache.shardingsphere.orchestration.temp.internal.registry.fixture.SecondTestRegistryCenter
+org.apache.shardingsphere.orchestration.internal.registry.fixture.FifthTestRegistryCenter
```

Comment:
Shall these three TestRegistryCenter be renamed as TestDistributedLockManagement?